cmd/nodecmd/create.go

```diff
@@ -217,40 +216,88 @@ func createNodes(_ *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-
-	time.Sleep(30 * time.Second)
-
+	if err := updateAnsiblePublicIPs(clusterName); err != nil {
+		return err
+	}
+	inventoryPath := app.GetAnsibleInventoryDirPath(clusterName)
 	avalancheGoVersion, err := getAvalancheGoVersion()
 	if err != nil {
 		return err
 	}
-	inventoryPath := app.GetAnsibleInventoryDirPath(clusterName)
 	if err = ansible.CreateAnsibleHostInventory(inventoryPath, cloudConfig.CertFilePath, cloudService, publicIPMap); err != nil {
 		return err
 	}
+	allHosts, err := ansible.GetInventoryFromAnsibleInventoryFile(inventoryPath)
+	if err != nil {
+		return err
+	}
+	hosts := ansible.FilterHostsByNodeID(allHosts, cloudConfig.InstanceIDs)
+	// waiting for all nodes to become accessible
+	failedHostMap := waitForHosts(hosts)
+	failedHosts := []string{}
+	for node, err := range failedHostMap {
+		ux.Logger.PrintToUser("Instance %s failed to provision with error %s. Please check instance logs for more information", node, err)
```

Comment:
is "to provision" clear regarding not beign able to connect to ssh?

Reply:
err will have this information